canonical-websites/www.ubuntu.com
# Created templates for engage case study and webinar heros

## Done

- Created templates for heros for engage pages for case studies and webinars

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- You will need to setup a test page or add one of the below includes on a page in the /engage directory to test these

### Case study headers

#### no image or class

```{% include "engage/shared/_header-case-study.html" with title="Allan Gray unlocks unprecedented availability and productivity with Ubuntu K8s"   %}```

html
```
<section class="p-strip--light">
  <div class="row u-equal-height">
    <div class="col-8">
      <h1>
        Allan Gray unlocks unprecedented availability and productivity with Ubuntu K8s
      </h1>
    </div>
  </div>
</section>
```

![image](https://user-images.githubusercontent.com/441217/49179238-cc447200-f349-11e8-94f3-7bd12afd6099.png)

#### image, no class

```{% include "engage/shared/_header-case-study.html" with title="Allan Gray unlocks unprecedented availability and productivity with Ubuntu K8s"   %}```

html

```
<section class="p-strip--light">
  <div class="row u-equal-height">
    <div class="col-7">
      <h1>
        Allan Gray unlocks unprecedented availability and productivity with Ubuntu K8s
      </h1>
    </div>
    <div class="col-5 u-align--center u-vertically-center">
      <img src="https://assets.ubuntu.com/v1/071489b4-K8s+logo+mobile.svg" alt="image for (( h1 ))">
    </div>
  </div>
</section>
```
![image](https://user-images.githubusercontent.com/441217/49179486-4d036e00-f34a-11e8-84ab-e1025558621a.png)


#### image, class

```{% include "engage/shared/_header-case-study.html" with title="Allan Gray unlocks unprecedented availability and productivity with Ubuntu K8s"  image="071489b4-K8s+logo+mobile.svg" class="test-class" %}```

html
```
<section class="p-strip test-class">
  <div class="row u-equal-height">
    <div class="col-7">
      <h1>
        Allan Gray unlocks unprecedented availability and productivity with Ubuntu K8s
      </h1>
    </div>
    <div class="col-5 u-align--center u-vertically-center">
      <img src="https://assets.ubuntu.com/v1/071489b4-K8s+logo+mobile.svg" alt="image for (( h1 ))">
    </div>
  </div>
</section>
```

![image](https://user-images.githubusercontent.com/441217/49179579-745a3b00-f34a-11e8-9218-eada70061718.png)

### Webinar headers

#### title, subtitle, date, cta, class

```
{% include "engage/shared/_header-webinar.html" with title="What's New in Ubuntu 18.10"  image="d9105909-Dell_XPS_Laptop_Left-Desktop.png?w=413" subtitle="Canonical product managers discuss the new Ubuntu 18.10 release features in this on-demand webinar" date="Webinar is live on 3 December 2018 at 8pm GMT" url="#register-section" cta="Watch the webinar" class="test-class" %}
```

```
<section class="p-strip test-class">
  <div class="row u-equal-height">
    <div class="col-7">
      <h1 style="font-weight: 100;">
        What's New in Ubuntu 18.10
      </h1>
      <h4>
        Canonical product managers discuss the new Ubuntu 18.10 release features in this on-demand webinar
      </h4>
      <p>
        Webinar is live on 3 December 2018 at 8pm GMT
      </p>
      <p>
        <a href="#register-section" class="p-button--neutral">
          Watch the webinar
        </a>
      </p>
    </div>
    <div class="col-5 u-align--center u-vertically-center">
      <img src="https://assets.ubuntu.com/v1/d9105909-Dell_XPS_Laptop_Left-Desktop.png?w=413" alt="image for (( title ))">
    </div>
  </div>
</section>
```
![image](https://user-images.githubusercontent.com/441217/49181028-36f7ac80-f34e-11e8-8429-3b1ca6ce734e.png)


#### no date

```
{% include "engage/shared/_header-webinar.html" with title="What's New in Ubuntu 18.10"  image="d9105909-Dell_XPS_Laptop_Left-Desktop.png?w=413" subtitle="Canonical product managers discuss the new Ubuntu 18.10 release features in this on-demand webinar" url="#register-section" cta="Watch the webinar" %}
```

```
<section class="p-strip--light">
  <div class="row u-equal-height">
    <div class="col-7">
      <h1 style="font-weight: 100;">
        What's New in Ubuntu 18.10
      </h1>
      <h4>
        Canonical product managers discuss the new Ubuntu 18.10 release features in this on-demand webinar
      </h4>
      
      <p>
        <a href="#register-section" class="p-button--neutral">
          Watch the webinar
        </a>
      </p>
    </div>
    <div class="col-5 u-align--center u-vertically-center">
      <img src="https://assets.ubuntu.com/v1/d9105909-Dell_XPS_Laptop_Left-Desktop.png?w=413" alt="image for (( title ))">
    </div>
  </div>
</section>
```
![image](https://user-images.githubusercontent.com/441217/49181071-50005d80-f34e-11e8-9a9c-081e6f2f878f.png)

#### no image, no class

```
{% include "engage/shared/_header-webinar.html" with title="What's New in Ubuntu 18.10" url="#register-section" cta="Watch the webinar"  %}
```

```
{% include "engage/shared/_header-webinar.html" with title="What's New in Ubuntu 18.10"  subtitle="Canonical product managers discuss the new Ubuntu 18.10 release features in this on-demand webinar" date="Webinar is live on 3 December 2018 at 8pm GMT" url="#register-section" cta="Watch the webinar" %}
```

```
<section class="p-strip--light">
  <div class="row u-equal-height">
    <div class="col-8">
      <h1 style="font-weight: 100;">
        What's New in Ubuntu 18.10
      </h1>
      <h4>
        Canonical product managers discuss the new Ubuntu 18.10 release features in this on-demand webinar
      </h4>
      <p>
        Webinar is live on 3 December 2018 at 8pm GMT
      </p>
      <p>
        <a href="#register-section" class="p-button--neutral">
          Watch the webinar
        </a>
      </p>
    </div>
    
  </div>
</section>
```
![image](https://user-images.githubusercontent.com/441217/49181124-71614980-f34e-11e8-8a3b-996811903595.png)

#### no subtitle, no image, no class

```
<section class="p-strip--light">
  <div class="row u-equal-height">
    <div class="col-8">
      <h1 style="font-weight: 100;">
        What's New in Ubuntu 18.10
      </h1>
      
      
      <p>
        <a href="#register-section" class="p-button--neutral">
          Watch the webinar
        </a>
      </p>
    </div>
    
  </div>
</section>
```
![image](https://user-images.githubusercontent.com/441217/49181163-8ccc5480-f34e-11e8-935c-376f5547c545.png)

## Issue / Card

Fixes #4404
